velox/dwio/dwrf/test/StripeReaderBaseTests.cpp

```diff
@@ -126,11 +127,5 @@ TEST_F(StripeLoadKeysTest, ThirdStripeHasKey) {
 }
 
 TEST_F(StripeLoadKeysTest, KeyMismatch) {
-  EXPECT_THAT(
-      [&]() { runTest(3); },
-      Throws<facebook::velox::dwio::common::exception::LoggedException>(
-          Property(
-              &facebook::velox::dwio::common::exception::LoggedException::
-                  failingExpression,
-              HasSubstr("keys.size() == providers_.size()"))));
+  VELOX_ASSERT_THROW(runTest(3), "keys.size() == providers_.size()");
```

Comment:
This fails the test with
```
[ RUN      ] StripeLoadKeysTest.KeyMismatch
/__w/velox/velox/velox/dwio/dwrf/test/StripeReaderBaseTests.cpp:130: Failure
Value of: e.message().find("keys.size() == providers_.size()") != std::string::npos
  Actual: false
Expected: true
Expected error message to contain 'keys.size() == providers_.size()', but received '[Range Constraint Violation : 2==1] : '.
[  FAILED  ] StripeLoadKeysTest.KeyMismatch (1 ms)
```
You'd think it should match like for the others.

Reply:
TIL that VeloxException has a `failingExpression` which is what is being tested here.
`Range Constraint Violation : 2==1` is the actual exception message but the failing expression is `keys.size() == providers_.size()`